rules/Php55/RegexMatcher.php

```diff
@@ -46,6 +46,10 @@ public function resolvePatternExpressionWithoutEIfFound(Expr $expr): Concat|Stri
                 return null;
```

Comment:
Won’t this not handle `(…)e` now?

Reply:
that seems require separate PR as the issue is on escaping no quote in `$0`:

```php
echo preg_replace('{[[:upper:]]+}e', '_$0', 'FOO');
```

which seems needs extra check.

Feel free to provide a different PR for solution for that if you found a solution first.